package.json

```diff
@@ -39,6 +39,7 @@
     "rm-dev-link": "rimraf -f \"$(yarn global bin)/amplify-dev\"",
     "setup-dev-win": "yarn dev-build && yarn link-win && yarn link-aa-win",
     "setup-dev": "yarn dev-build && yarn rm-dev-link && yarn link-dev && yarn rm-aa-dev-link && yarn link-aa-dev",
+    "sort-packages": "yarn eslint --no-eslintrc --config .eslint.package.json.js **/package.json --fix || echo 'The dependencies in `package.json` are not sorted correctly, please run yarn sort-packages to fix this issue.'",
```

Comment:
If we're passing `--fix` why would we `echo` ? Should echo actually go to `lint-check-package-json` here https://github.com/aws-amplify/amplify-cli/blob/d0ffcc53112efc22d138bf52eecd40e37f98ce8d/package.json#L30

Reply:
This is what I was intending to happen. When a user commits something that is not correct, have the error message guide them in CI.

Reply:
I will address this in a new PR.